app/controllers/application_controller.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
     skip_before_action :verify_authenticity_token
+    rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+
+    def record_not_found(exception)
+        render json: {status: "error", code:404, message: "Record not found" }, status: :not_found
+    end
```

Comment:
We will be handling our not found mainly on our frontend side so this may not be completely necessary for our API

Reply:
Perfect! wanted to have something in place now, we can refactor it out once we have frontend taken care of :-)